src/javascript/SelectorTypes/DateTimePicker/DateTimePicker.jsx

```diff
@@ -49,7 +85,8 @@ DateTimePicker.propTypes = {
     id: PropTypes.string.isRequired,
     editorContext: PropTypes.shape({
         lng: PropTypes.string,
```

Comment:
Not part of this commit, but this look like a typo, could be great to fix it or remove the prop if it's not used.

Reply:
Done.